src/lsp-extensions.ts

```diff
@@ -0,0 +1,39 @@
+import * as lsp from 'vscode-languageserver/node';
```

Comment:
We have a `lsp-protocol.calls.proposed.ts` file for the propesed `textDocument/calls` implementation so I think it would make sense to follow that naming convention and call this one `lsp-protocol.inlayHints.proposed.ts` instead of `lsp-extensions.ts`.